srcpkgs/cargo-deny/template

```diff
@@ -1,23 +1,32 @@
 # Template file for 'cargo-deny'
 pkgname=cargo-deny
-version=0.12.2
+version=0.13.5
 revision=1
+_adv_commit="222ccf84ef7d733a6eac71f694a5e14b4d0d0289"
 build_style=cargo
 configure_args="--no-default-features"
 hostmakedepends="pkg-config"
 makedepends="openssl-devel"
+checkdepends="git"
 short_desc="Cargo plugin to help you manage large dependency graphs"
 maintainer="Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
 license="MIT, Apache-2.0"
 homepage="https://github.com/EmbarkStudios/cargo-deny"
 changelog="https://raw.githubusercontent.com/EmbarkStudios/cargo-deny/main/CHANGELOG.md"
 distfiles="https://github.com/EmbarkStudios/cargo-deny/archive/${version}.tar.gz"
-checksum=7d8a034fe0219f8b7adc0ab25c90f7f253b4c387d7d5c8f74f6ad79f4810dc1e
+checksum=aebfff46cf510fc9a85ce0059e88792c6467f44c17dd29badce50a13af47e4f7
 
 if [ "$XBPS_TARGET_ENDIAN" = "be" ]; then
 	broken="bitvec crate unimplemented for big endian"
 fi
 
+pre_check() {
+	rm -rf tests/advisory-db/github.com-2f857891b7f43c59
+	git clone https://github.com/rustsec/advisory-db.git tests/advisory-db/github.com-2f857891b7f43c59
+	cd tests/advisory-db/github.com-2f857891b7f43c59
+	git checkout ${_adv_commit}
+}
```

Comment:
We should probably fetch advisory-db as a distfile

Reply:
Doesn't work, it wants a git checkout for the tests

Reply:
Ah, nevermind then